scss/_patterns_lists.scss

```diff
@@ -73,25 +77,49 @@
 }
 
 // Displays a list inline with spacing
-@mixin vf-lists-inline-list {
-  .inline-list {
+@mixin vf-p-lists-inline-list {
+  .p-inline-list {
     margin-left: 0;
+    padding-left: 0;
 
-    li {
-      display: inline;
-      list-style: none;
-      margin-right: 20px;
+    &__item {
+      @extend %inline-list-item;
     }
+  }
+}
 
-    .last-item {
-      margin-right: 0;
+// Displays a list inline with items separated by dots
+@mixin vf-p-lists-inline-list-middot {
+  .p-inline-list--middot {
+    margin-left: 0;
+    padding-left: 0;
+
+    .p-inline-list__item {
+      @extend %inline-list-item;
+      position: relative;
+
+      &::after {
+        content: '•';
+        color: $color-mid-grey;
+        line-height: 0;
+        font-size: 1.4rem;
+        position: absolute;
+        right: -1rem;
+        top: .55rem;
+      }
+
+      &:last-of-type,
+      .last-item {
+        &::after {
+          content: '';
+        }
+      }
     }
   }
 }
 
 // Displays a step by step list of instructions
-@mixin vf-stepped-list {
-
+@mixin vf-p-stepped-list {
```

Comment:
We're using the more natural language ordering with other classes - like "inline-list" - and it stops it being mistaken for a modifier this way.